blackbox/docs/config/session.rst

```diff
@@ -57,7 +60,10 @@ Supported Session Settings
      includes it in the ``search_path`` *before* the configured schemas, unless
      it is already explicitly in the schema configuration.
 
-``enable_semijoin``: *enabled* (default: ``false``)
+**enable_semijoin**
+  | *Default:* ``false``
+  | *Modifiable:* ``yes``
+
   An :ref:`experimental <experimental-warning>` setting which enables CrateDB
```

Comment:
Suggestion:
Since we have more categories now (e.g. `modifiable`), 
does it make sense to add another one for `experimental` for make this more explicit?

thoughts?

Reply:
we discussed it, leaving as it is for now.